ADBench/run-all.ps1

```diff
@@ -51,7 +51,11 @@ param(# Which build to test.
       [double]$time_limit=10,  
       
       # Kill the test after this many seconds 
-      [double]$timeout=300,    
+      [double]$timeout=300,
+
+      # Kill the test if it consumes more than this many gigabytes of RAM.
+      # If parameter value is less or equal to zero, RAM checking isn't performed.
+      [double]$max_memory_amount_in_gb=-1,
```

Comment:
Does PowerShell have something you could use in the same way that one would use `None` in Python?  i'm not particularly comfortable storing "no limit" in-band as a negative number.

Reply:
While '0 means unlimited' is a quite traditional convention used in many programs, there is, as 
@awf mentioned, `$null`. However, if we're going for non-numbers, I'd like to add `[double]::PositiveInfinity` to consideration. It has a very direct "no limit" meaning and allows to use the same code for both limited and unlimited scenarios.

Reply:
Yes, I think that's the best solution.

Reply:
I agree. It would allow us to remove the check `$max_memory_amount_in_gb -gt 0` completely.

Reply:
I've added using positive infinity instead of negative numbers. So, I think, this issue is solved